protected/humhub/modules/comment/Module.php

```diff
@@ -83,7 +84,7 @@ public function canComment($object)
 
         $content = $object->content;
 
-        if ($content->container instanceof Space) {
+        if ($content->container) {
             $space = $content->container;
             if (!$space->permissionManager->can(CreateComment::class)) {
```

Comment:
I would just change the line here:
```php
 if (!$content->container->permissionManager->can(CreateComment::class)) {
 ```

Reply:
Note that if `intsanceof Space` isn't removed then the permissions don't set right for User permissions on their posts for profile stream, this is why it's still removed in this p/r